src/Umbraco.Core/Models/ContentBase.cs

```diff
@@ -264,7 +264,17 @@ public virtual bool HasProperty(string propertyTypeAlias)
         /// <returns><see cref="Property"/> Value as an <see cref="object"/></returns>
         public virtual object GetValue(string propertyTypeAlias)
         {
-            return Properties[propertyTypeAlias].Value;
+            try

```

Comment:
@warrenbuckley Instead use `Properties.TryGetValue` which is much more efficient and is the correct way to do this. Try/Catch should only be used when absolutely necessary.